storm-client/test/jvm/org/apache/storm/TestConfigValidate.java

```diff
@@ -84,21 +83,6 @@ public void invalidConfigTest() throws InvocationTargetException, NoSuchMethodEx
         ConfigValidation.validateFields(conf);
     }
 
-    @Test(expected = InvalidTopologyException.class)
```

Comment:
I tried to mock it and then decided not to because the logic is really simple in `validateTopologyBlobStoreMap`. But I will add the unit test if you think it's better